app/actions.js

```diff
@@ -23,7 +22,25 @@ const client = new Client({
   maxFeeds: MAX_FEEDS,
   config: {
     dbdir: DATA_DIR
-  }
+  },
+  commands: {
+    help: {
+      help: () => 'display this help message',
+      call: (cabal, res, arg) => {
+        const commands = client.getCommands()
+        let helpContent = ''
+        for (var key in commands) {
+          helpContent = helpContent + `/${key} - ${commands[key].help()} \n`
+        }
+        addStatusMessage({ addr: cabal.key, text: helpContent })
+      }
+    }
+  }  
+})
+// Disable a few slash commands for now
+const removedCommands = ['add', 'channels', 'clear', 'ids', 'names', 'new', 'qr', 'whoami', 'whois']
+removedCommands.forEach((command) => {
+  client.removeCommand(command)
```

Comment:
➕

Reply:
yep those commands need a signal from cabal-client that they're happening and we can add them back in when we can